### ISSUE TITLE
Creating an index without the need of a class

### ISSUE THREAD

**Issue by wouterv** [orientechnologies/orientdb-lucene#27]
I would like to be able to create an index referencing records, not limited to a single class, using multiple keys. 

Neo4j supports this with their legacy indexing functionality (http://neo4j.com/docs/stable/indexing.html).

So for example, to create such an index, do something like
```
create index index_name ('id','name') FULLTEXT ENGINE LUCENE
```
Then be able to insert like
```
insert into index:index_name (id, name, rid) values (1, 'jeff', #45:0)
insert into index:index_name (id, name, rid) values (1, 'Luca', #45:0)
insert into index:index_name (id, name, rid) values (2, 'Luca', #45:1)
```
and then query stuff like
```
select * from index:index_name where [id, name] LUCENE "(id:1 OR name:Luca)"
```

Not sure if orientdb needs to be changed for this, or that this can be added to this project.

**Comment by lvca** [comment]
This issue could be done together with "Search Engine" issue

**Comment by maggiolo00** [comment]
Don't think will be ready for 2.1
Can

**Comment by maggiolo00** [comment]
Has been implemented in 2.1.x
Here it is the docs
http://orientdb.com/docs/last/Full-Text-Index.html#create-a-manual-lucene-index

**Comment by Ziink** [comment]
Although this does implement a manual Lucene index, it misses the mark by not allowing naming the fields.

So it appears that there is no way to get what wouterv suggested in the first post to work.

It would be nice to be able to query the manual lucene index similar to what wouterv suggested 

```sql
select * from index:index_name where [id, name] LUCENE "(id:1 OR name:Luca)"
```
